stable/artifactory-ha/templates/_helpers.tpl

```diff
@@ -0,0 +1,46 @@
+{{/* vim: set filetype=mustache: */}}
+{{/*
+Expand the name of the chart.
+*/}}
+{{- define "artifactory.name" -}}
+{{- default .Chart.Name .Values.nameOverride | trunc 63 | trimSuffix "-" -}}
+{{- end -}}
+
+{{/*
+The primary node name
+*/}}
+{{- define "artifactory.primary.name" -}}
+{{- default "artifactory-primary" .Values.artifactory.primary.name | trunc 63 | trimSuffix "-" -}}
+{{- end -}}
+
+{{/*
+The member node name
+*/}}
+{{- define "artifactory.node.name" -}}
+{{- default "artifactory-node" .Values.artifactory.node.name | trunc 63 | trimSuffix "-" -}}
+{{- end -}}
+
+{{/*
+Expand the name nginx service.
+*/}}
+{{- define "artifactory.nginx.name" -}}
+{{- default .Values.nginx.name .Values.nameOverride | trunc 63 | trimSuffix "-" -}}
+{{- end -}}
+
+{{/*
+Create a default fully qualified app name.
+We truncate at 63 chars because some Kubernetes name fields are limited to this (by the DNS naming spec).
+*/}}
+{{- define "artifactory.fullname" -}}
```

Comment:
Please use the latest fullname template you get with `helm create` and Helm 2.8+.

Reply:
Will update